PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/common/classifiedexception/SparkUserException.kt

```diff
@@ -35,7 +35,8 @@ object SparkUserExceptionFactory : ClassifiedExceptionFactory() {
     override fun createClassifiedException(exp: Throwable?): ClassifiedException? {
         return if (exp is YarnDiagnosticsException
                 // Throw from wrong class name ,refer to issue 1827 and 2466
-                || exp is SparkJobException) {
+                || exp is SparkJobException
+                || exp is IllegalArgumentException) {
```

Comment:
Not handle `IllegalSecuryKeyException`?

Reply:
if key is invalid ,the container is null which handles in running configuration